command_test.go

```diff
@@ -0,0 +1,158 @@
+// Copyright 2016 Keybase, Inc. All rights reserved. Use of
+// this source code is governed by the included BSD license.
+
+package updater
+
+import (
+	"os"
+	"reflect"
+	"testing"
+	"time"
+
+	"github.com/keybase/go-logging"
+)
+
+var log = logging.Logger{Module: "test"}
+
+func TestEmptyRunCommand(t *testing.T) {
+	out, err := RunCommand("", nil, time.Second, log)
+	if out != "" {
+		t.Errorf("Unexpected output: %s", out)
```

Comment:
Please use [testify](/stretchr/testify) to make asserts. This also trivializes JSON testing because it allows you to directly [assert JSON equality](https://godoc.org/github.com/stretchr/testify/require#JSONEq)